code/modules/clothing/modular_armor/attachments.dm

```diff
@@ -196,7 +201,7 @@
 
 /obj/item/armor_module/armor/Initialize()
 	. = ..()
-	item_state = initial(icon_state) + "_a"
+	item_state = base_icon + "_a"
```

Comment:
i dont believe `item_state` does anything here. just delete this line.
(test though because it could just be me forgetting stuff)

Reply:
done and tested